src/Context/authContext.jsx

```diff
@@ -0,0 +1,28 @@
+import {createContext, useContext, useState}  from 'react';
+
+const AuthContext = createContext();
+
+const AuthProvider = ({children}) => {
+
+    const getToken = localStorage.getItem("login-token");
+    const getLocalStorageUser = localStorage.getItem("user");
+    const [token, setToken] = useState(getToken || "")
+    const [user, setUser] = useState(JSON.parse(getLocalStorageUser) || "")
```

Comment:
Suggestion to make it more logical you could have used Nullish Coalescing operator (??)